tools/tpp-run/tpp-run.cpp

```diff
@@ -91,19 +91,37 @@ llvm::cl::opt<unsigned>
 // Target Triple
 // Default x86_64, can be changed to aarch64 on other arches
 llvm::cl::opt<std::string> triple("triple", llvm::cl::desc("Target triple"),
+#if defined(__x86_64__)
                                   llvm::cl::init("x86_64-linux-gnu"));
+#elif defined(__aarch64__)
+                                  llvm::cl::init("aarch64-linux-gnu"));
+#else
+#error Unsupported architecture
+#endif
 
 // Target CPU name
 // Default skylake is old enough to be relevant for most cases
 llvm::cl::opt<std::string>
     cpuName("cpu", llvm::cl::desc("CPU name (sapphirerapids, alderlake, etc)"),
+#if defined(__x86_64__)
             llvm::cl::init("skylake"));
+#elif defined(__aarch64__)
+            llvm::cl::init("cortex-a53"));
```

Comment:
Is this is pretty old aarch64 CPU, such that everything works on Graviton, M1/M2 etc?

Reply:
It's the baseline. This is just the default in case we don't pass anything. Code generated for A53 will work in pretty much on every v8/v9 core.

For GVT we may want to pass the argument to define the right core, to make use of SVE when available. But we don't want to rely on non-TPP code-gen for these benchmarks anyway, so it should be fine anyway.